notebooks/Update_TCGA_Data_Model_dataguids_to_Compact_DRS_Identifier.ipynb

```diff
@@ -0,0 +1,236 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <center>Update TCGA Workspace Data Model with Compact DRS URLs/Identifier</center>\n",
+    "\n",
+    "##### Description\n",
+    "    This notebook allows a user to update single entity (ex. participant, sample, and pair) data model tables from a  \"drs://dataguids.org\" file path, denoting the location of a file, to the newer compact DRS URL - drs://dg.4DFC:UUID - format. The notebook will isolate eligible columns that point to a file location with the dataguids.org pointer, create a new updated tsv file, and update the data model.\n",
```

Comment:
the names and descriptions of these notebooks were pretty confusing to me - so similar and the differences weren't clear. with so much overlap between them, could they be one notebook?